### ISSUE TITLE
Unhandled Promise Rejection Error: Image could not be created from /opt/ExifCleaner/exifcleaner.png | .deb package error

### ISSUE THREAD

**Issue by airvue** [szTheory/exifcleaner#80]
Unable to launch latest ExifCleaner on Linux distro, Pop!_OS 20.04 LTS (Ubuntu based)

`Error Message:`
```
Unhandled Promise Rejection
Error: Image could not be created from /opt/ExifCleaner/exifcleaner.png
    at createMainWindow (/opt/ExifCleaner/resources/app.asar/webpack:/src/main/window_setup.ts:74:9)
    at setup (/opt/ExifCleaner/resources/app.asar/webpack:/src/main/index.ts:23:14)
```
`Screenshot:`
![exifcleaner](https://user-images.githubusercontent.com/53982860/90925693-8abdeb00-e3a6-11ea-870c-1d383146dc97.png)

**Comment by Goblin80** [comment]
The problem is caused by the absence of `exifcleaner.png` image from `/opt/ExifCleaner/` which is used in the Help -> About ExifCleaner.

A temporary cheap fix would be to copy any small PNG image into the `/opt/ExifCleaner` directory and call it `exifcleaner.png`.

**Comment by szTheory** [comment]
Looking into this now.

**Comment by szTheory** [comment]
I've fixed the issue on master but need more time to test this new release. Hopefully it will be ready in a few days. Until then you could try a previous release or use the workaround @Goblin80 mentioned.

**Comment by szTheory** [comment]
Just a quick update. Sorry but I've been too busy to finish acceptance testing across all platforms. This next release contains a lot of changes and I want to get it right. It might take a while, but I'll let you know when it's ready.

**Comment by airvue** [comment]
No worries, understandable. Look forward to the fix!

Just a heads up, I tried the workaround suggested by @Goblin80 , but it did not solve the issue. No error message is given now, but no app launches.

**Comment by Goblin80** [comment]
> No worries, understandable. Look forward to the fix!
> 
> Just a heads up, I tried the workaround suggested by @Goblin80 , but it did not solve the issue. No error message is given now, but no app launches.

@airvue Are you sure about that? I noticed that the first launch after the fix doesn't register, but subsequent launches should work just fine.

**Comment by airvue** [comment]
@Goblin80 

![exif](https://user-images.githubusercontent.com/53982860/92539785-5dbb6600-f1f7-11ea-9c1f-67d673bc50ae.gif)

**Comment by Goblin80** [comment]
@airvue huh, weird.

Are you sure you are using a PNG 8-bit/color RGBA, non-interlaced image?

**Comment by airvue** [comment]
@Goblin80 , never mind, your workaround fixed the issue:
![cleaner](https://user-images.githubusercontent.com/53982860/92828964-7f982280-f388-11ea-828e-16403f3a8d3c.gif)
Thanks, the gif before this was using an 8-bit colormap, not RGBA, PNG

One issue that stands is that the desktop icon does not appear, however I am familiar with fixing that.

**Comment by minutesinch** [comment]
in which version get patched?

**Comment by opp** [comment]
> @Goblin80 , never mind, your workaround fixed the issue:
> ![cleaner](https://user-images.githubusercontent.com/53982860/92828964-7f982280-f388-11ea-828e-16403f3a8d3c.gif)
> Thanks, the gif before this was using an 8-bit colormap, not RGBA, PNG
> 
> One issue that stands is that the desktop icon does not appear, however I am familiar with fixing that.

how did you get the window to open? i've followed the same process you did and it doesn't show error message anymore but the window still doesn't pop up. i also have a PNG 8-bit/color RGBA, non-interlaced image and even found one that was 50x50 in case that made a difference, lol.

**Comment by szTheory** [comment]
@airvue @Goblin80 @FredyAmaya @minutesinch I've just released a new version 3.4.0 that should fix this if you want to give it a try https://github.com/szTheory/exifcleaner/releases/tag/v3.4.0

**Comment by airvue** [comment]
@szTheory , exifcleaner v3.4.0 .deb works on my end, thanks!
